<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="sabariPc" failures="3" tests="4" name="BankingApp Test(failed)" time="86.201" errors="0" timestamp="16 Aug 2022 15:22:11 GMT">
  <testcase classname="org.inetBanking.testCases.TC_LoginDDT_002" name="loginDDT" time="2.306">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.inetBanking.testCases.TC_LoginDDT_002.loginDDT(TC_LoginDDT_002.java:29)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- loginDDT -->
  <testcase classname="org.inetBanking.testCases.TC_LoginDDT_002" name="loginDDT" time="17.739">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.inetBanking.testCases.TC_LoginDDT_002.loginDDT(TC_LoginDDT_002.java:29)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- loginDDT -->
  <testcase classname="org.inetBanking.testCases.TC_LoginDDT_002" name="loginDDT" time="3.038">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.inetBanking.testCases.TC_LoginDDT_002.loginDDT(TC_LoginDDT_002.java:29)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- loginDDT -->
  <testcase classname="org.inetBanking.testCases.TC_LoginTest_001" name="loginTest" time="23.823"/>
</testsuite> <!-- BankingApp Test(failed) -->
